File: HurManager.Dto/Houses/HouseSummary.cs
namespace HurManager.Dto.Houses
{
    public class HouseSummary
    {
        public int Id { get; set; }

        public string Address { get; set; }

        public int? WaterMeterId { get; set; }

        public int WaterMeterReading { get; set; }

        public string WaterMeterFactoryNumber { get; set; }
    }
}

File: HurManager.App/client-app/src/models/HouseSummary.ts
export class HouseSummary {
    id: number = 0;
    address: string = "";
    waterMeterId: number = 0;
    waterMeterReading: number = 0;
    waterMeterFactoryNumber: string = "";

    constructor(init?: Partial<HouseSummary>) {
        if (init) {
            Object.assign(this, init);
        }
    }
}
